helm/idol-nifi/templates/zookeeper/configmap.yaml

```diff
@@ -8,12 +8,15 @@
 # The information contained herein is subject to change without notice.
 #
 # END COPYRIGHT NOTICE
+{{- range $nifiClusterItem := .Values.nifiClusters }}
```

Comment:
Is there any need for a separate ConfigMap per cluster if they're all the same data?